docs/vue/slides.md

```diff
@@ -40,7 +39,7 @@ vue upgrade --next
 Once that is done, install the Swiper dependency in your project:
 
 ```shell
-npm install swiper@8
+npm install swiper@latest
 ```
 
 ## Swiping with Style
```

Comment:
Since we opted not to merge https://github.com/ionic-team/ionic-framework/pull/26942, shouldn't these guides have the "Swiping with Style" section updated to manually add the styles instead of importing our stylesheet?

Reply:
I assumed that only applied to the new Swiper Element styles (i.e. Angular only), but I'm open to also deprecating the existing `ionic-slides.scss` if desired.

Reply:
Oh I see - React/Vue don't use the Web Component. In that case let's keep it the same. Whenever we update the React/Vue guide to use Swiper Element we can remove the `ionic-slides.scss` reference.